### PR TITLE
Simplify SDK

### PR DESCRIPTION
**Pull request by haiodo**
Simplify SDK

Fixes related to https://github.com/networkservicemesh/networkservicemesh/pull/1524

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

**Comment by nickolaev** [comment]
Looks good. It will fail of course until we merge the main SDK PR in the nsm repo

File: .circleci/config.yml
Comment (nickolaev) [inline .circleci/config.yml]: Not needed.

File: .circleci/config.yml
Comment (nickolaev) [inline .circleci/config.yml]: Not needed.

File: .circleci/config.yml
Comment (nickolaev) [inline .circleci/config.yml]: Not needed.

File: examples/universal-cnf/vppagent/cmd/config/composite.go
Comment (nickolaev) [inline examples/universal-cnf/vppagent/cmd/config/composite.go:67]: Hmm, should that be `return conn, nil`?

File: examples/universal-cnf/vppagent/cmd/config/config_endpoints.go
Comment (nickolaev) [inline examples/universal-cnf/vppagent/cmd/config/config_endpoints.go:56]: Should these be swapped?

File: examples/universal-cnf/vppagent/cmd/config/config_endpoints.go
Comment (nickolaev) [inline examples/universal-cnf/vppagent/cmd/config/config_endpoints.go:69]: Hmm, is this still appending?

**Review by nickolaev — COMMENTED** [review]
Looks good. Have some questions though. Please check.

File: examples/universal-cnf/vppagent/cmd/config/composite.go
Comment (haiodo) [reply examples/universal-cnf/vppagent/cmd/config/composite.go:67]: It is same, but it would be cleaner to return conn. Thanks.

File: examples/universal-cnf/vppagent/cmd/config/config_endpoints.go
Comment (haiodo) [reply examples/universal-cnf/vppagent/cmd/config/config_endpoints.go:56]: Nope, Monitor the only part was not reversed, since it send update to monitor only after all other parties are complete.

File: examples/universal-cnf/vppagent/cmd/config/config_endpoints.go
Comment (haiodo) [reply examples/universal-cnf/vppagent/cmd/config/config_endpoints.go:69]: Not sure I've understand comment.

**Review by nickolaev — APPROVED** [review]
The failures look strange and might be related to some CI resource scheduling. Will merge this and try to fix after.

**Pull request merged**
